lib/i18n/hygiene/keys_with_script_tags.rb

```diff
@@ -7,7 +7,7 @@ module Hygiene
     class KeysWithScriptTags
       include Enumerable
 
-      SCRIPT_TAG_REGEX = /<script>.*<\/script>/
```

Comment:
Do we need to look for a closing tag at all? Is there a risk if a phrase includes a `<script>` tag and doesn't close it?